app/javascript/javascript/params.js

```diff
@@ -5,5 +5,31 @@ export const getParams = () =>
     .filter(x => x.length !== 0)
     .map(x => x.split("="))
     .map(([k, v]) => [k, decodeURIComponent(v)])
-    .map(([k, v]) => [k, JSON.parse(v)])
+    .map(([k, v]) => [k, parseIfJSON(v)])
```

Comment:
Because some of our params are like `data = {a:1, b:2}` but some are like `theme = default`. So in the latter case you can't really do `JSON.parse("default")`.

But hopefully, this is just a temporary solution, like we discussed maybe we will remove the `data` params